CI/appveyor.after_success.ps1

```diff
@@ -71,31 +71,39 @@ if ("$Env:APPVEYOR_REPO_TAG" -eq "false" -and -Not $Script:PublicTestBuild) {
 
   $Script:NuSpec = "C:\projects\installers\windows\mudlet.nuspec"
   Write-Output "=== Creating Nuget package ==="
+  # the last change (and the only one in the else branch) can be removed when we
+  # can arrange for the old value "mudlet_main_512x512_6XS_icon.ico" to be
+  # replaced in the mudlet/installer repository:
   if ($Script:PublicTestBuild) {
     # allow public test builds to be installed side by side with the release builds by renaming the app
     # no dots in the <id>: https://github.com/Squirrel/Squirrel.Windows/blob/master/docs/using/naming.md
     (Get-Content "$Script:NuSpec").replace('<id>Mudlet</id>', '<id>Mudlet-PublicTestBuild</id>') | Set-Content "$Script:NuSpec"
     (Get-Content "$Script:NuSpec").replace('<title>Mudlet</title>', '<title>Mudlet (Public Test Build)</title>') | Set-Content "$Script:NuSpec"
+    (Get-Content "$Script:NuSpec").replace('<iconUrl>https://raw.githubusercontent.com/Mudlet/Mudlet/development/src/icons/mudlet.ico</iconUrl>', '<title>https://raw.githubusercontent.com/Mudlet/Mudlet/development/src/icons/mudlet_ptb.ico</iconUrl>') | Set-Content "$Script:NuSpec"
+    (Get-Content "$Script:NuSpec").replace('<iconUrl>https://raw.githubusercontent.com/Mudlet/Mudlet/development/src/icons/mudlet_main_512x512_6XS_icon.ico</iconUrl>', '<title>https://raw.githubusercontent.com/Mudlet/Mudlet/development/src/icons/mudlet_ptb.ico</iconUrl>') | Set-Content "$Script:NuSpec"
+  } else {
+    (Get-Content "$Script:NuSpec").replace('<iconUrl>https://raw.githubusercontent.com/Mudlet/Mudlet/development/src/icons/mudlet_main_512x512_6XS_icon.ico</iconUrl>', '<title>https://raw.githubusercontent.com/Mudlet/Mudlet/development/src/icons/mudlet.ico</iconUrl>') | Set-Content "$Script:NuSpec"
   }
   nuget pack "$Script:NuSpec" -Version "$Script:VersionAndSha" -BasePath $SQUIRRELWIN -OutputDirectory $SQUIRRELWIN
 
   Write-Output "=== Creating installers from Nuget package ==="
   if ($Script:PublicTestBuild) {
-    $TestBuildString = "-PublicTestBuild"
-    $InstallerIconFile = "${Env:APPVEYOR_BUILD_FOLDER}\src\icons\mudlet_ptb.ico"
+    $nupkg_path = "C:\projects\squirrel-packaging-prep\Mudlet-PublicTestBuild.$Script:VersionAndSha.nupkg"
```

Comment:
IIRC it was because the `TestBuildString` and  `InstallerIconFile` variables were only used in one place and it was a bit easier to keep track of which file was used in which build at the point of use rather than a previous setup point. I would have done the same with `nupkg_path` except that it is used in more than one place.

Reply:
I think it's easier to understand when things are in variables like that